lib/repository/interface/genre_interface.dart

```diff
@@ -0,0 +1,3 @@
+abstract class GenreInterface {
+  Future<Map<int, String>> getGenre();
```

Comment:
I'd rename this as `getGenres`, as you're returning several genres

Reply:
Done